test/specialty.jl

```diff
@@ -238,6 +238,14 @@ using ImageFiltering: IdentityUnitRange
         fwhm = ceil(Int, (α*2*sqrt(2^(1/β) - 1)) * 4)
         @test Kernel.moffat(α, β) == Kernel.moffat(α, β, (fwhm, fwhm))
     end
+
+    @testset "butterworth" begin
+        a = rand()
+        b = rand()
+        @test Kernel.butterworth(a,b,(3,3)) == Kernel.butterworth(a,b,3)
+        @test Kernel.butterworth(a,b,(4,4)) == Kernel.butterworth(a,b,4)
```

Comment:
We also need some numerical tests here, can you test against one or two results generated from other frameworks that has `butterworth` implemented or manual calculated results? Also do remember to add a comment on how the references are generated.

We also need to test that the kernel axis are "centered". For instance, the axes should be `(-1:1, -1:1)` when kernel size is `(3, 3)`. Also need to test the even case as well.